_fixtures/sleep.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
We have a fixture that sleeps indefinitely which we could use for this, we could just rename that file. It's `_fixtures/test1145.go`.